changes/1424-atheuz.md

```diff
@@ -0,0 +1 @@
+include format with secret types #546
```

Comment:
Please remove the issue id, you could use it in the file name instead of the pr Id if you like, but we shouldn't have both.

Please use back quotes around `format`.

Please add "to the schema"